pages/linux/tlp.md

```diff
@@ -0,0 +1,27 @@
+# tlp
+
+> Advanced Power Management for Linux.
+
+- Apply settings (according to the actual power source):
+
+`sudo tlp start`
+
+- Apply battery settings (_ignoring_ the actual power source):
```

Comment:
@agnivade / @waldyrious, is the italics here ok? I haven't come across this before.

Reply:
I don't see a grammatical need for italics. Better to remove it.